starter-code/index.js

```diff
@@ -1,11 +1,54 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
-  };
+
+    constructor() {
+    this.items = [];
+    this.length = 0;
+    }
+
+    add(item) {
+      this.items.push(item);
+      this.length += 1;
+    }
+    
+    get(pos) {
+      return this.items[pos];
+    }
```

Comment:
Ici ça fonctionne, mais il est également demandé de renvoyer une erreur si la position demandée n'existe pas dans l'array. Par exemple si on demande l'élément à l'index 10, alors qu'il n'y a que 3 éléments dans l'array. On pourrait écrire :
```javascript
	  if (pos > this.length) {
		throw new Error('OutOfBounds');
	  }
```
dans la fonction get, avant `return this.items[pos];`